### ISSUE TITLE
Module explodes in stack trace

### ISSUE THREAD

**Issue by SimenB** [davglass/license-checker#34]
Tried using this, it exploded though. Node 0.12.2 npm 2.9.1.

```
TypeError: Cannot read property 'indexOf' of null
    at /Users/simen/.nvm/versions/node/v0.12.2/lib/node_modules/license-checker/lib/index.js:80:36
    at Array.forEach (native)
    at flatten (/Users/simen/.nvm/versions/node/v0.12.2/lib/node_modules/license-checker/lib/index.js:76:11)
    at /Users/simen/.nvm/versions/node/v0.12.2/lib/node_modules/license-checker/lib/index.js:104:20
    at Array.forEach (native)
    at flatten (/Users/simen/.nvm/versions/node/v0.12.2/lib/node_modules/license-checker/lib/index.js:95:40)
    at /Users/simen/.nvm/versions/node/v0.12.2/lib/node_modules/license-checker/lib/index.js:104:20
    at Array.forEach (native)
    at flatten (/Users/simen/.nvm/versions/node/v0.12.2/lib/node_modules/license-checker/lib/index.js:95:40)
    at /Users/simen/.nvm/versions/node/v0.12.2/lib/node_modules/license-checker/lib/index.js:118:20
```

Just clone my repo https://github.com/SimenB/webpack-fun to reproduce.

Edit. Version 2.0.2 works fine though

**Comment by davglass** [comment]
Pushed a fix in `license-checker@3.0.2`

**Comment by SimenB** [comment]
@davglass Works great now, thanks!
